en/docs/SetupAndInstall/DeployingWSO2APIManager/ThirdPartyKeyManager/configuring-wso2-identity-server-as-a-key-manager.md

```diff
@@ -237,8 +237,8 @@ The following instructions are only applicable in the following scenarios:
         ...
     </APIManager>
     ```
-    -   If you are working with a **single** **Key Manager** in hybrid single node setup where WSO2 IS is the Key Manager and the rest of the API-M components are in one node, you need to replace `            {IS-server-host}           ` with the actual host of the WSO2 IS sever **** node.
-    -   If you are working with a **single** **Key Manager** in distributed setup, you need to replace `            {IS-server-host}           ` with the actual host of the WSO2 IS sever **** node.
+    -   If you are working with a **singleKey Manager** in hybrid single node setup where WSO2 IS is the Key Manager and the rest of the API-M components are in one node, you need to replace `            {IS-server-host}           ` with the actual host of the WSO2 IS sever **** node.
```

Comment:
Space is missing.

Reply:
done

Reply:
Changes in the configuring-wso2-identity-server-as-a-key-manager.md file are not available in the commit on Sep 9, 2019

Reply:
The fix is available in the commit on Sep 10, 2019 - https://github.com/wso2/docs-apim/pull/19/commits/8866914ed8e436a24bae938b18fe042f0322eda